### ISSUE TITLE
(5.0.12) Request for separate player options button from start button

### ISSUE THREAD

**Issue by cgoudie** [stepmania/stepmania#1263]
The start button is overloaded with starting the game. It's a crummy user experience for a new player who clicks start, and then accidentally clicks it again, to then navigate that menu.

I would like to propose the ability to configure a different pad button for player options, and if configured, that button is used in place of the long hold or double press on start.

**Comment by kyzentun** [comment]
The problem I see with this is every old user would object to the change, unless the old behavior was default so they didn't notice.  With the old behavior as default, new users wouldn't benefit at all.

I agree that going through the options menu sucks for players that didn't intend it.  That's why my custom options screen puts "Play Song" at the top instead of the bottom.

**Comment by cgoudie** [comment]
@kyzentun I agree, putting play at the top would be an improvement. I believe in practice *most* users don't want to see the options screen at all; only the advanced user that wants to tune their options.
